piety-promoted-v1/modernized/17-chapter-16.adoc

```diff
@@ -761,18 +766,23 @@ When he was visited with the sickness of which he died,
 which continued on him about a quarter of a year,
 he bore it with much patience and resignation to the will of God, and very cheerfully;
 which was comfortable,
-both to his parents and friends who visited him in the time of his illness.`'
+both to his parents and friends who visited him in the time of his illness.
 
 Before his departure,
 he was concerned to advise and counsel his brother and sister to fear and love God,
 and be dutiful to their parents, etc.
 And though it is our loss, to have such a one taken from us,
 we believe it is his gain to be removed from where trouble and temptations attend,
-to where the wicked cease from troubling, and the weary are at rest.`'
+to where the wicked cease from troubling, and the weary are at rest.
 
-From our meeting in Dublin, the 19th of the Second month 1709.`'
+[.signed-section-context-close]
```

Comment:
This is a bit weird. Normally this SSCC would be BELOW the signature, and not above it. I think it would look best to bump this down below the signature part.